lib/fileset_picker.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/python
+# Copyright 2015 Google.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import encoder
+import glob
+import mpeg_settings
+import optimizer
+import os
+
+
+class Error(Exception):
+  pass
+
+
+def ChooseRates(width, framerate):
+  if width >= 1920 and framerate >= 50:
+    return [3000, 4500, 7000, 10000]
+  if width >= 1920 and framerate >= 24:
+    return [1600, 2500, 4000, 6000]
+  if width >= 832 and framerate >= 50:
+    return [512, 768, 1200, 2000]
+  if width >= 416:
+    return [384, 512, 850, 1500]
+  raise Error('Unhandled width/framerate combo: w=%d rate=%d' %
+              (width, framerate))
+
+
+def GenerateFileset(name):
+  yuvfiles = glob.glob(os.path.join(os.getenv('WORKDIR'),
```

Comment:
I cringe a bit at having important stuff like working directories as environment variables that get read throughout the code in various places. It can be painful to pass paths from the top level down to where they are used, but it's also easier to reason about which paths are used. Using env variables at the top level like a main()) which then calls down is fine though.